colabs/Wczytywanie_danych_NASA_w_sparku.ipynb

```diff
@@ -0,0 +1,262 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
```

Comment:
To mi się nie podoba
`
from pyspark.sql.functions import col
from pyspark.sql.types import IntegerType, FloatType
for column in df.columns:
    if column == "Date":
        df = df.withColumn(column, col(column).cast(IntegerType()))
    else:
        df = df.withColumn(column, col(column).cast(FloatType()))
`
W sparku lepiej to zrobić definiując schemat typów przy wczytywaniu. Poprawmy, albo załóżmy osobne zadanie na poprawienie tego w szablonie - bo wiem, że zadanie mocno blokuje innych.